api/turing/cluster/knative_service.go

```diff
@@ -118,12 +119,16 @@ func (cfg *KnativeService) buildSvcSpec(
 		"autoscaling.knative.dev/class":    autoscalingMetricClassMap[cfg.AutoscalingMetric],
 	}
 
+	if cfg.InitialReplicas != nil {
+		annotations["autoscaling.knative.dev/initial-scale"] = strconv.Itoa(*cfg.InitialReplicas)
```

Comment:
nit: Since the annotation is `initial-scale` I think it's better to rename `InitialReplicas` to `InitialScale` to avoid more confusion and make it consistent across Turing components.